testdata/xml-validate/invalid-data.xml

```diff
@@ -0,0 +1,76 @@
+<?xml version='1.0' encoding='utf-8'?>
+
+<knora xmlns="https://dasch.swiss/schema"
+       xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xsi:schemaLocation="https://dasch.swiss/schema https://raw.githubusercontent.com/dasch-swiss/dsp-tools/main/src/dsp_tools/resources/schema/data.xsd"
+       shortcode="9999"
+       default-ontology="onto">
+
+    <!-- Cardinality Violation -->
+
+    <resource label="Max Cardinality Violation" restype=":SubClass" id="max-card-violation">
+        <text-prop name=":hasSimpleText">
+            <text encoding="utf8">Text Value One</text>
+            <text encoding="utf8"
+                  comment="This property has a cardinality 1"
+            >Text Value Two</text>
```

Comment:
```suggestion
            <text encoding="utf8">Text Value Two</text> <!-- Violation: Property has cardinality 1, but two values were supplied -->
```

I think you misunderstood my suggestion of "using comments for indicating what the violation is", sorry about that.